docs/developer-docs/setup/playground.md

```diff
@@ -102,6 +102,55 @@ git clone https://github.com/dfinity/motoko-playground
 }
 ```
 
+## Creating a custom playground with separate pool creation
+
+As an alternative to a staging environment or the Motoko mainnet playground, the Motoko playground allows for custom, private playgrounds to be deployed. Using a custom playground allows for extensive customization, such as enabling access control by restricting the playground's usage to only allow certain principals, configuring more generous canister timeouts and the amount of available cycles, and allowing some (or all) of the function calls that the mainnet Motoko playground does not allow, such as sending cycles to other canisters. 
+
+Using a custom playground can help simplify development for teams, since the whole team can use a custom playground without needing to manage individual cycle balances. To create a custom playground, **separate pool creation** can be used. 
```

Comment:
```suggestion
Using a custom playground can help simplify development for teams, since the whole team can use a custom playground without needing to manage individual cycle balances.
```
I probably explained this badly. Leaving this out is just fine